part-4/model-building-components.md

```diff
@@ -279,7 +279,10 @@ Tags are evaluated in fields of observers, transports, or event groups titled "I
 
```

Comment:
Conditions can be combined using either **AND logic** (`Condition1 AND Condition2 AND ...`), or **OR logic** (`Condition1 OR Condition2 OR ...`).